apps/wallpaper/style/wallpaper.css

```diff
@@ -22,16 +22,20 @@ html, body {
 }
 
 .wallpaper {
-  border-bottom: 0.1rem solid #000000;
-  border-right: 0.1rem solid #000000;
   position: relative;
   overflow: hidden;
   float: left;
   width: 33%;
+  -moz-margin-end: 0.1rem;
```

Comment:
Fixes first two columns shows up flushed against each other in RTL mode. Favor using -margin-end over explicitly specifying border-left in RTL mode.